tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -3847,25 +3879,39 @@ Duration <DURATION>"
 `;
 
 exports[`simulator_permissions.test.w 1`] = `
-"error: Expected a preflight object as first argument to \`lift\` builtin, found inflight expression instead
-   --> ../../../examples/tests/invalid/simulator_permissions.test.w:10:8
+"error: Expected expression
+   --> ../../../examples/tests/invalid/simulator_permissions.test.w:10:21
    |
 10 |   lift(buckets.at(1), [\\"put\\"]);
-   |        ^^^^^^^^^^^^^ Expected a preflight object as first argument to \`lift\` builtin, found inflight expression instead
+   |                     ^ Expected expression
 
 
-error: Expression of type \\"Bucket\\" references an unknown preflight object, can't qualify its capabilities. Use \`lift()\` to explicitly qualify the preflight object to disable this error.
-   --> ../../../examples/tests/invalid/simulator_permissions.test.w:15:3
+error: Expected expression
+   --> ../../../examples/tests/invalid/simulator_permissions.test.w:20:21
    |
-15 |   buckets.at(i).put(\\"key\\", \\"value\\");
-   |   ^^^^^^^^^^^^^ Expression of type \\"Bucket\\" references an unknown preflight object, can't qualify its capabilities. Use \`lift()\` to explicitly qualify the preflight object to disable this error.
+20 |   lift(buckets.at(0), [\\"list\\"]);
+   |                     ^ Expected expression
+
+
+error: Unknown parser error
+   --> ../../../examples/tests/invalid/simulator_permissions.test.w:10:3
+   |
+10 |   lift(buckets.at(1), [\\"put\\"]);
+   |   ^^^^ Unknown parser error
 
 
-error: Expected a preflight object as first argument to \`lift\` builtin, found inflight expression instead
-   --> ../../../examples/tests/invalid/simulator_permissions.test.w:20:8
+error: Unknown parser error
+   --> ../../../examples/tests/invalid/simulator_permissions.test.w:20:3
    |
 20 |   lift(buckets.at(0), [\\"list\\"]);
-   |        ^^^^^^^^^^^^^ Expected a preflight object as first argument to \`lift\` builtin, found inflight expression instead
+   |   ^^^^ Unknown parser error
+
+
+error: Expression of type \\"Bucket\\" references an unknown preflight object, can't qualify its capabilities. Use \`lift()\` to explicitly qualify the preflight object to disable this error.
```

Comment:
Can we add a hint to this diagnostic that links to the language reference section? I imagine here is a good place: https://www.winglang.io/docs/concepts/inflights#explicit-lift-qualification